packages/react-components/src/organisms/PastEventsDashboardCard.tsx

```diff
@@ -64,7 +69,14 @@ const PastEventsDashboardCard: React.FC<PastEventsDashboardCardProps> = ({
         ({ id, title, startDate, notes, presentation, videoRecording }) => (
           <div key={id} css={[rowStyles]}>
             <span css={[titleStyles, rowTitleStyles]}>Event</span>
-            <Link ellipsed href={eventsRoute({}).event({ eventId: id }).$}>
+            <Link
```

Comment:
I wonder if it wouldn't be better to have something like

```
{linksEnabled ? (
  <Link ellipsed href={eventsRoute({}).event({ eventId: id }).$}>
    {title}
  </Link>
) : (
  <span>{title}</span>
)}
```

because with the `Link` the title is still blue and has an underline on hover so it gives the impression that can be clickable

Reply:
Yea, that sounds better. Updating...